customer-testing/src/main/java/no/unit/nva/customer/testing/CustomerDataGenerator.java

```diff
@@ -101,17 +102,24 @@ public static CustomerDao createSampleCustomerDao() {
                                    .withDoiAgent(randomDoiAgent(randomString()))
                                    .withNviInstitution(randomBoolean())
                                    .withSector(randomSector())
+                                   .withInactiveFrom(randomInstant())
                                    .withRightsRetentionStrategy(randomRightsRetentionStrategy())
                                    .withAllowFileUploadForTypes(randomAllowFileUploadForTypes())
                                    .build();
         assertThat(customer, doesNotHaveEmptyValues());
         return customer;
     }
 
+    public static CustomerDao createSampleActiveCustomerDao() {
```

Comment:
Lager en egen for aktiv og en egen for inaktiv kunde. (flere randomCustomerGenerators rundt om kring).

Dette slik at sjekket at rundturen gjennom serialisering får med seg alle verdiene.
Samtidig så vil de fleste logiske operasjoner og sjekker skje på aktive kunder.

å sette inactiveFrom til å være return randomBolea() ? randomInstant() : null vil føre til at tester tryner randomly.